book/src/contributing/using-dev-container.md

```diff
@@ -0,0 +1,55 @@
+# Development in Dev Containers
+
+```admonish note
+This document is a heavy work in progress and may not contain sufficient information.
+```
```

Comment:
I applaud the approach of disclaiming when something isn't complete, and merging quickly rather than waiting until everything is perfect.

But is there a specific thing these docs lack? To me this looks fairly complete! 

Disclaimers like this should either a) be specific, so we know the next thing to do, or b) have someone's name attached, so they can own getting it to the next stage. Otherwise it's very likely to be there for a long time...